x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/use_find_attack_discoveries/index.ts

```diff
@@ -18,6 +18,7 @@ import { useCallback, useRef } from 'react';
 import { useKibanaFeatureFlags } from '../use_kibana_feature_flags';
 
 interface Props {
+  alertIds?: string[];
```

Comment:
This PR is not tagged as being backported to `8.19`, but consider backporting this change to simplify future backports and testing, (avoiding diverging implementations).